examples/chat/weaver_gen.go

```diff
@@ -997,6 +1009,136 @@ func (s sQLStore_server_stub) getImage(ctx context.Context, args []byte) (res []
 	return enc.Data(), nil
 }
 
+// Reflect stub implementations.
+
+type imageScaler_reflect_stub struct {
+	caller func(reflect.Type, string, []reflect.Value) []reflect.Value
+}
+
+// Check that imageScaler_reflect_stub implements the ImageScaler interface.
+var _ ImageScaler = (*imageScaler_reflect_stub)(nil)
+
+func (s imageScaler_reflect_stub) Scale(ctx context.Context, a0 []byte, a1 int, a2 int) (r0 []byte, err error) {
+	component := reflect.TypeOf((*ImageScaler)(nil)).Elem()
```

Comment:
Could we reduce the size of generated code. Perhaps something like:

```
results, err := s.caller("Scale", []any{a0, a1, a2})
return results[0].([]byte), err
```

I am assuming the component type (ImageScaler here) is already baked into the s.caller closure.

Reply:
Great idea! I updated the code with your suggestion. There's no longer any use of `reflect.Value`, so let me know if you think we should call these stubs something different than "reflect stubs".